app/views/home/index.html.erb

```diff
@@ -1,4 +1,8 @@
 <% name = current_user.present? ? current_user.first_name + ' ' + current_user.last_name : 'Stranger' %>
 
 <h1>Welcome, <%= name %> </h1>
-<p>Find me in app/views/home/index.html.erb</p>
\ No newline at end of file
+<% if current_user.present? %>
```

Comment:
We should probably not have this screen at all, we should simply have login screen when the user first goes to the website.

Reply:
okay so we need change the route definition for redirecting only signed users to home screen. (next iteration)